sparse/unit_test/Test_Sparse_StaticCrsGraph.hpp

```diff
@@ -0,0 +1,260 @@
+  using dView              = Kokkos::StaticCrsGraph<ordinal_type, layout_type, space_type, memory_traits_type>;
+  using hView              = typename dView::HostMirror;
+
+  dView dx;
+
+  // StaticCrsGraph with Unmanaged trait will contain row_map and entries
+  // members with the Unmanaged memory trait. Use of such a StaticCrsGraph
+  // requires an allocaton of memory for the unmanaged views to wrap.
+  //
+  // In this test, a graph (via raw arrays) resides on the host.
+  // The pointers are wrapped by unmanaged Views.
+  // To make use of this on the device, managed device Views are created
+  // (allocation required), and data from the unmanaged host views is deep
+  // copied to the device Views Unmanaged views of the appropriate type wrap the
+  // device data and are assigned to their corresponding unmanaged view members
+  // of the unmanaged StaticCrsGraph
+
+  // Data types for raw pointers storing StaticCrsGraph info
+  using ptr_row_map_type = typename dView::size_type;
+  using ptr_entries_type = typename dView::data_type;
+
+  const ordinal_type numRows = 8;
+  const ordinal_type nnz     = 24;
+  ptr_row_map_type ptrRaw[]  = {0, 4, 8, 10, 12, 14, 16, 20, 24};
+  ptr_entries_type indRaw[]  = {0, 1, 4, 5, 0, 1, 4, 5, 2, 3, 2, 3, 4, 5, 4, 5, 2, 3, 6, 7, 2, 3, 6, 7};
+
+  // Wrap pointers in unmanaged host views
+  using local_row_map_type = typename hView::row_map_type;
+  using local_entries_type = typename hView::entries_type;
+  local_row_map_type unman_row_map(&(ptrRaw[0]), numRows + 1);
+  local_entries_type unman_entries(&(indRaw[0]), nnz);
+
+  hView hx;
+  hx = hView(unman_entries, unman_row_map);
+
+  // Create the device Views for copying the host arrays into
+  // An allocation is needed on the device for the unmanaged StaticCrsGraph to
+  // wrap the pointer
+  using d_row_map_view_type = typename Kokkos::View<ptr_row_map_type*, layout_type, space_type>;
+  using d_entries_view_type = typename Kokkos::View<ptr_entries_type*, layout_type, space_type>;
+
+  d_row_map_view_type tmp_row_map("tmp_row_map", numRows + 1);
+  d_entries_view_type tmp_entries("tmp_entries", nnz);
+
+  Kokkos::deep_copy(tmp_row_map, unman_row_map);
+  Kokkos::deep_copy(tmp_entries, unman_entries);
+
+  // Wrap the pointer in unmanaged View and assign to the corresponding
+  // StaticCrsGraph member
+  dx.row_map = typename dView::row_map_type(tmp_row_map.data(), numRows + 1);
+  dx.entries = typename dView::entries_type(tmp_entries.data(), nnz);
+
+  ASSERT_TRUE((std::is_same_v<typename dView::row_map_type::memory_traits, Kokkos::MemoryUnmanaged>));
+  ASSERT_TRUE((std::is_same_v<typename dView::entries_type::memory_traits, Kokkos::MemoryUnmanaged>));
+  ASSERT_TRUE((std::is_same_v<typename hView::row_map_type::memory_traits, Kokkos::MemoryUnmanaged>));
+  ASSERT_TRUE((std::is_same_v<typename hView::entries_type::memory_traits, Kokkos::MemoryUnmanaged>));
+}
+
+} /* namespace TestStaticCrsGraph */
+
+TEST(NotSureWhatGoesThere, staticcrsgraph) {
```

Comment:
Need to fix this before we merge
Let me know how you would like to call it

Reply:
We just put `TestCategory` and it gets set in e.g. test_common/Test_HIP.hpp for different backends.

Reply:
Looks like this still needs to be addressed, adding Carl's advice as a code suggestion:

```suggestion
TEST(TestCategory, staticcrsgraph) {
```